css/main.css

```
@@ -8,3 +8,84 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+.beyonce{
+
+
+}
+
+.perfilbeyonce{
```

Separar o nomes com traços, tipo:
```suggestion
.perfil-beyonce{
```